src/Uno.UWPSyncGenerator/Program.cs

```diff
@@ -33,6 +33,9 @@ static void Main(string[] args)
 				new SyncGenerator().Build(@"..\..\..\Uno.UWP", "Uno", "Windows.Networking.Connectivity.WwanContract");
 				new SyncGenerator().Build(@"..\..\..\Uno.UWP", "Uno", "Windows.ApplicationModel.Calls.CallsPhoneContract");
 
+				// When adding support for a new WinRT contract here, ensure to add it to the list of origins in Generator.cs
+				// and to the list of supported contracts in ApiInformation.shared.cs
```

Comment:
Consider a `Debug.Fail` for the `default` case in ApiInformation.IsApiContractPresent